### PR TITLE
New Directory Structure

### PR DESCRIPTION
**Pull request by joelshawaddepar**
Updating directory structure to break apart `core` style from `onyx` specific styling. Here are some notes about the changes:

- Created a new directory for all `core` variables and styling. 
- Created a new directory for all `onyx` variables and styling. 
- Cleaned up font directory
- Starting to introduce some of the react styling for inputs
- Added a bunch of README files to help explain various sections
- Starting to introduce new typography (Created `/onyx/utilities/typography/index-iverson.scss` to make sure new typography doesn't impact Iverson) @LydiaBarnes, let me know what you think about prefixing files for specific use cases. Ideally this will not be done often, however, if we need a custom set of files we can create a custom `index-[custom-use-case].scss` to import or not import any specific files.

### CODE REVIEW

File: styles/core/utilities/index.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//  -----------------------------------
+//  Core - Utilities | Index
+//  -----------------------------------
+
+@import 'core-functions/index';
```
Comment (LydiaBarnes) [inline styles/core/utilities/index.scss]: This naming seems a bit redundant, its already in core folder.

**Comment by joelshawaddepar** [comment]
Alright, I got a bunch of stuff removed. 

2 - I removed a bunch of stuff here. As far as colors go, I broke that into a few files and created a new directory to simplify importing all color variables. 

3/4 -  Makes sense, got this simplified.

**Comment by LydiaBarnes** [comment]
I still see a bunch of new stuff in here, like all that input stuff you added and onyx typography. What I meant was keep this PR to just renaming/reshuffling of _existing_ files.
For any new code, we need to analyze each element/component on an individual level (thus in separate PRs) instead of adding all at once. Esp for adding anything to the core section. There is no rush to getting everything in style-toolbox.

It may be easier for you to create a new branch/PR just for naming, rather than pull out of this one.

**Comment by LydiaBarnes** [comment]
As an aside, simplifying this to just naming this is esp important bc I need to make more changes to some existing code, and that will cause conflicts, so I need this naming stuff done soon. Small PRs makes that way easier.

**Comment by joelshawaddepar** [comment]
addressed in part here: https://github.com/Addepar/addepar-style-toolbox/pull/19

**Pull request closed without merging**